src/textbooks/hooks.jsx

```diff
@@ -0,0 +1,93 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useContext, useEffect } from 'react';
+import { AppContext } from '@edx/frontend-platform/react';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { useToggle } from '@openedx/paragon';
+
+import { updateSavingStatus } from './data/slice';
+import { RequestStatus } from '../data/constants';
+import {
+  getTextbooksData,
+  getLoadingStatus,
+  getSavingStatus,
+} from './data/selectors';
+import {
+  createTextbookQuery,
+  fetchTextbooksQuery,
+  editTextbookQuery,
+  deleteTextbookQuery,
+} from './data/thunk';
+import messages from './messages';
+
+const useTextbooks = (courseId) => {
+  const intl = useIntl();
+  const dispatch = useDispatch();
+  const { config } = useContext(AppContext);
+
+  const textbooks = useSelector(getTextbooksData);
+  const loadingStatus = useSelector(getLoadingStatus);
+  const savingStatus = useSelector(getSavingStatus);
+
+  const [isTextbookFormOpen, openTextbookForm, closeTextbookForm] = useToggle(false);
+
+  const breadcrumbs = [
+    {
+      label: intl.formatMessage(messages.breadcrumbContent),
+      href: `${config.STUDIO_BASE_URL}/course/${courseId}`,
+    },
+    {
+      label: intl.formatMessage(messages.breadcrumbPagesAndResources),
+      href: `/course/${courseId}/pages-and-resources`,
+    },
+    {
+      label: '',
+      href: `/course/${courseId}/pages-and-resources/textbooks`,
```

Comment:
This seems to be the wrong URL.  I could only get it to work (in a Tutor dev environment) with:

```suggestion
      href: `/course/${courseId}/textbooks`,
```

Reply:
Fixed

Reply:
this is right url after refactor routes